Dockerfile

```diff
@@ -30,12 +30,14 @@ COPY --parents .yarn/ yarn.lock .yarnrc.yml **/package.json packages/bind-mount/
 #
 # If the following issue is ever addressed, we can use that instead:
 # https://github.com/yarnpkg/berry/issues/6339
-RUN cd /PrairieLearn && yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
+WORKDIR /PrairieLearn
```

Comment:
This is a nice change, this means that one will already be in this directory when the container starts!

Can we move this towards the top of the file so that the big comment above this is directly adjacent to the `yarn install` below?

Reply:
Fixed in 5ae834225dd4c7b110bee80a810331ddd36ee375.